toolchain/check/class.h

```diff
@@ -0,0 +1,32 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_TOOLCHAIN_CHECK_CLASS_H_
+#define CARBON_TOOLCHAIN_CHECK_CLASS_H_
+
+#include "toolchain/check/context.h"
+
+namespace Carbon::Check {
+
+// If `type_id` is a class type, get its corresponding `SemIR::Class` object.
+// Otherwise returns `nullptr`.
+auto TryGetAsClass(Context& context, SemIR::TypeId type_id) -> SemIR::Class*;
+
+// Builds the `Self` type using the resulting type constant.
+auto SetNewClassSelfTypeId(Context& context, SemIR::ClassId class_id) -> void;
+
+// Tracks that this declaration is the definition and introduce `Self`.
+auto TrackClassDefinition(Context& context, SemIR::ClassId class_id,
+                          SemIR::InstId class_decl_id) -> SemIR::Class&;
+
+// Compute object representation for a fully defined class.
```

Comment:
```suggestion
// Computes the object representation for a fully defined class.
```

Style: "Function comments should be written with an implied subject of _This function_" https://google.github.io/styleguide/cppguide.html#Function_Comments

So, "This function computes the object ...", not "This function compute object ..."

Reply:
Done.